README.md

```diff
@@ -87,6 +87,94 @@ Example:
 java -jar refactoring.jar <dataset> <git-url> <output-path> <database-url> <database-user> <database-password> <k-threshold>
 ```
 
+## Database Clean-up
+
+The enormous variety refactoring types, projects and programming styles in the mined repositories can lead to various issues with the data. Therefore, we explain two common problems and potential solutions here.
+
+**!MAKE A COPY OF YOUR DATA BEFORE ALTERING IT!**
+
+### Remove unfinished projects
```

Comment:
Have you thought of putting the 1 project in a transaction so that if it fails at least unfinished projects don't need manual intervention? I would be willing to give this a go.

Reply:
We could do that, additionally.
But the warning is for unintended changes to your db, because you might remove many samples and regret it later.